app/uk/gov/hmrc/claimvatenrolmentfrontend/services/JourneyService.scala

```diff
@@ -38,20 +39,24 @@ class JourneyService @Inject()(journeyConfigRepository: JourneyConfigRepository,
     } yield id
   }
 
-  def retrieveJourneyConfig(journeyId: String, authInternalId: String): Future[JourneyConfig] =
+  def retrieveJourneyConfig(journeyId: String, authInternalId: String)(implicit request: Request[_]): Future[Option[JourneyConfig]] =
     journeyConfigRepository.retrieveJourneyConfig(journeyId, authInternalId).map {
       case Some(journeyConfig) =>
-        journeyConfig
+        infoLog(s"[JourneyService][retrieveJourneyConfig] - successfully retrieved the journey config: $journeyConfig")
```

Comment:
logging the journey id instead of the config might be more useful for monitoring. The journeyConfig just contains the continueUrl